tests/operators/test_finite_difference_op.py

```diff
@@ -0,0 +1,53 @@
+"""Tests for finite difference operator."""
+
+# Copyright 2024 Physikalisch-Technische Bundesanstalt
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import torch
+from mrpro.operators import FiniteDifferenceOp
+
+from tests import RandomGenerator
+from tests.helper import dotproduct_adjointness_test
+
+
+@pytest.mark.parametrize('mode', ['central', 'forward', 'backward'])
+def test_finite_difference_op_forward(mode):
+    """Test correct finite difference of simple object."""
+    # Test object with linear gradient in real and imaginary part
+    linear_gradient_object = torch.arange(1, 21)[None, :].to(dtype=torch.float32)
+    linear_gradient_object = linear_gradient_object + 1j * linear_gradient_object
```

Comment:
Small suggestion: do it with different direction in real and image part. So we would notice some mixup there.

Reply:
And maybe with more than onenaxis in the input.